src/modelmanager.cpp

```diff
@@ -225,6 +226,62 @@ Status ModelManager::loadPipelinesConfig(rapidjson::Document& configJson) {
     return ovms::StatusCode::OK;
 }
 
+Status ModelManager::loadCustomLoadersConfig(rapidjson::Document& configJson) {
+    const auto itrp = configJson.FindMember("custom_loader_config_list");
+    if (itrp == configJson.MemberEnd() || !itrp->value.IsArray()) {
+        return StatusCode::OK;
+    }
+
+    // Load Customer Loaders as per the configuration
+    SPDLOG_INFO("Using Customloaders");
+    auto& customloaders = ovms::CustomLoaders::instance();
+    for (const auto& configs : itrp->value.GetArray()) {
+        const std::string loaderName = configs["config"]["loader_name"].GetString();
+        SPDLOG_INFO("Reading Custom Loader:{} configuration", loaderName);
+
+        CustomLoaderConfig loaderConfig;
+        auto status = loaderConfig.parseNode(configs["config"]);
+        if (status != StatusCode::OK) {
+            SPDLOG_ERROR("Parsing loader:{} config failed", loaderName);
+            return status;
+        }
+
+        SPDLOG_INFO("CHECK IF LOADER IS ALREADY LOADED");
+        if (customloaders.find(loaderName) == nullptr) {
```

Comment:
I take it we could extract L250-278 as a function. Not sure of name but clearly those lines do something else so we could divide this as a separate abstraction.
processCustomLoaderConfig?

Reply:
Made this as a new function inside ModelManager class